exchanges/kucoin/kucoin_test.go

```diff
@@ -3025,37 +3025,40 @@ func TestSubscribeBatches(t *testing.T) {
 func TestSubscribeBatchLimit(t *testing.T) {
 	t.Parallel()
 
+	const expectedLimit = 400
+
 	ku := testInstance(t) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
 	ku.Features.Subscriptions = subscription.List{}
 	testexch.SetupWs(t, ku)
 
 	avail, err := ku.GetAvailablePairs(asset.Spot)
 	require.NoError(t, err, "GetAvailablePairs must not error")
 
-	err = ku.CurrencyPairs.StorePairs(asset.Spot, avail[:299], true)
+	err = ku.CurrencyPairs.StorePairs(asset.Spot, avail[:expectedLimit-1], true)
```

Comment:
How come this is - 1? Works fine without it and still fills all the pair buckets 🤷

Reply:
I think previously it was erroring *at* 300.
Now it's erroring at 400, so I agree, we can raise it.

Fixed https://github.com/gbjk/gocryptotrader/commit/4a48e48a7e